src/components/APIStructurePreview.tsx

```diff
@@ -94,23 +94,23 @@ interface CardProps {
   position: React.CSSProperties;
 }
 
-const Card = (props: CardProps) => (
-  <article
-    ref={props.innerRef}
-    id="structures-tooltip"
-    role="tooltip"
-    className={clsx('alert', 'alert--info', 'shadow--md', styles.card)}
-    style={props.position}
-  >
-    <div className="card__body">
-      <ReactMarkdown
-        allowedElements={['h1', 'ul', 'li', 'code', 'a', 'em']}
```

Comment:
I wonder if we can retain this behavior, to ensure we don't introduce a regression. [Here's the relevant code](https://github.com/remarkjs/react-markdown/blob/9eb589e828445916dfb521117040d8d5420a5e9d/lib/index.js#L258-L278) in `react-markdown`, which simplifies when you exclude the options we wouldn't use (`allowElement` and `unwrapDisallowed`).

Reply:
Good point. Here's a regression I found for example:

- https://electronjs.org/docs/latest/api/web-contents/#contentsgetprintersasync
- https://am-i-cooking-chat.electron-website.pages.dev/docs/latest/api/web-contents/#contentsgetprintersasync

Reply:
[2e44fbb](https://github.com/electron/website/pull/690/commits/2e44fbb4819d0e99d505ae331d02ee8574ebc14b)